Support add configuration for plugin (#60)
apache/skywalking-go: docs/en/agent/plugin-configurations.md

# Plugin Configurations

| key                            | environment key                                       | default value | description                                                      |
|--------------------------------|-------------------------------------------------------|---------------|------------------------------------------------------------------|

> | http.server_collect_parameters | SW_AGENT_PLUGIN_CONFIG_HTTP_SERVER_COLLECT_PARAMETERS | false         | Collect the parameters when receive http request on server side. |

Reviewer: ```suggestion
| http.server_collect_parameters | SW_AGENT_PLUGIN_CONFIG_HTTP_SERVER_COLLECT_PARAMETERS | false         | Collect the parameters of the HTTP request on the server side. |
```
